cobalt/site/docs/reference/starboard/modules/player.md

```diff
@@ -449,6 +551,24 @@ Returns whether the given player handle is valid.
 static bool SbPlayerIsValid(SbPlayer player)
 ```
 
+### SbPlayerSeek
+
+`seek_to_timestamp`: The frame at which playback should begin. `ticket`: A user-
+supplied unique ID that is be passed to all subsequent
```

Comment:
"that is be" => "to be"

In general the comment of the function should be refined.  Please sync with @borongc to who's working on it to prioritize this.

Reply:
The PR (https://github.com/youtube/cobalt/pull/2958) is in progress, will modify `player.h` later.